Auth0/CredentialsManager.swift

```diff
@@ -29,7 +29,7 @@ import LocalAuthentication
 /// Credentials management utility
 public struct CredentialsManager {
 
-    private let storage = A0SimpleKeychain()
+    internal let storage: A0SimpleKeychain
```

Comment:
Which change to internal? The configured keychain should be passed in via the init?

Reply:
access to this property is changed back to private in the latest commit.

Reply:
Observing the storage property in needed to test that it is indeed using a bespoke keychain

Reply:
Ideally we could use a mock, although no protocol here. I would move this to it's original state of private and test against your custom instance.